### ISSUE TITLE
Read multiple YAML files in the same python file

### ISSUE THREAD

**Issue by Ferdinanddb** [ajbosco/dag-factory#122]
Hello,

This is not really an issue but more a question, I am wondering if you would recommend to have the following : instead of having one python file that would read one yaml file and construct a DAG, you construct multiple DAGs in the same python file by reading several YAML files.

From what I saw, the bottleneck would be at the worker level because it also has to parse the DAG to understand what it needs to do, and thus in this situation the worker would parse all the YAML files while it only needed to parse one. I did not find out if there is a way to stipulate that information to the airflow worker to not having it to parse something it doesn't need.

Regards
Ferdinand

**Comment by pankajastro** [comment]
Yes, we can read all `yaml` from a folder here is docs: https://astronomer.github.io/dag-factory/latest/configuration/load_yaml_dags/#1-loading-from-yaml-files-in-a-folder
